roles/confluent.schema_registry/templates/schema-registry_kerberos_ssl.properties.j2

```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-kafkastore.bootstrap.servers={% for inventory_hostname in groups['broker'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ inventory_hostname }}:{{broker.config.port}}{% endfor %}
+kafkastore.bootstrap.servers={% for inventory_hostname in groups['kafka'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ inventory_hostname }}:{{kafka_port}}{% endfor %}
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.